templates/plugin/dhcp_remote_isc.yml.erb

```diff
@@ -0,0 +1,30 @@
+---
+#
+# Configuration file for remote ISC dhcp provider
+#
+
+:config: <%= scope.lookupvar("::foreman_proxy::plugin::dhcp::remote_isc::dhcp_config") %>
+:leases: <%= scope.lookupvar("::foreman_proxy::plugin::dhcp::remote_isc::dhcp_leases") %>
+
+# Redhat 5
+#
+#:config: /etc/dhcpd.conf
+#
+# Settings for Ubuntu
+#
+#:config: /etc/dhcp3/dhcpd.conf
+#:leases: /var/lib/dhcp3/dhcpd.leases
+
+# Specifies TSIG key name and secret
+<% unless [nil, :undefined, :undef, ''].include?(scope.lookupvar("::foreman_proxy::plugin::dhcp::remote_isc::key_name")) ||
```

Comment:
Should there be a check in the class that if either is given, both should be given?